app/resources/css-default.template

```diff
@@ -1275,3 +1275,7 @@ table.api-keys td.empty {
   background: rgba(0,0,0,0.5);
   display: none;
 }
+
+.fb-share-button {
```

Comment:
Can you add a comment like /* Share buttons */? Otherwise it looks like a part of "Side menu" section.